docs/OCP_CI_Tutorials/Containers/Container_Creation_Guide.md

```diff
@@ -90,23 +100,27 @@ CMD ["/bin/bash"]
 ```
 
 ### Executing Containerized Tests in OpenShift CI
+
 Using a container is very easy in OpenShift CI. There are several ways it can be done (from a registry, a Dockerfile, an image stream, _etc._). However, for the purposes of this guide, we will focus on using the Dockerfile.
 
 #### Define the Container
+
 To define a container in OpenShift CI, two stanzas need to be added to the configuration file in OpenShift CI. Keep in mind, the configuration files in OpenShift CI are named based on the organization, repository, and branch of the test repository. So when we specify the Dockerfile's location or the context of the build, it is relative to the root of the testing repository. 
 
 Pretend we have a test repository in the `CSPI-QE` organization named `mock_tests` and we'd like to use the `main` branch of the repository to test with. The configuration file in the `openshift/release` repository would be `ci-operator/config/cspi-qe/mock_tests/cspi-qe-mock_tests-main.yaml`. 
```

Comment:
Since the organization name is case sensitive then we should use CSPI-QE instead of cspi-qe

Reply:
```suggestion
Pretend we have a test repository in the `CSPI-QE` organization named `mock_tests` and we'd like to use the `main` branch of the repository to test with. The configuration file in the `openshift/release` repository would be `ci-operator/config/CSPI-QE/mock_tests/CSPI-QE-mock_tests-main.yaml`. 
```
@madunn is this what you are looking for? I'm not sure if the file name is also case-sensitive.